modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/EntityCacheImpl.java

```diff
@@ -260,10 +258,27 @@ public void removeResult(Class<?> clazz, Serializable primaryKey) {
 		_removeResult(clazz, primaryKey, null);
 	}
 
+	public class CacheRegistryItemImpl implements CacheRegistryItem {
```

Comment:
Hey @GabrielHenriquedev, I think you only need to create this inner class in FinderCacheImpl, because it seems so be the same thing here, could you take a look at it? thx.